{{cookiecutter.project_slug}}/check.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -ex
+
+EXIT_STATUS=0
+
+# Autoformatter *first*, to avoid double-reporting errors
+black --check setup.py {{cookiecutter.package_name}} \
+    || EXIT_STATUS=$?
+
+# Run flake8 without pycodestyle and import-related errors
+flake8 {{cookiecutter.package_name}}/ \
+    --ignore=D,E,W,F401,F403,F405,F821,F822\
+    || EXIT_STATUS=$?
+
+# Uncomment to run mypy (to check static typing)
```

Comment:
We may want to mention that tests-requirements.txt also needs an uncomment?

Reply:
(Feel free to merge with or without adressing this.)

Reply:
`CHEATSHEET` has the whole list of things to touch if you want mypy.